snap/snapcraft.yaml

```diff
@@ -169,8 +168,16 @@ parts:
             #! /bin/bash
             if [ "\$1" == "xtest" ]
             then
-                xtest --install-ta \$SNAP_COMMON/lib/optee_armtz
-                xtest
+                if [ "\$2" == "--install-ta" ]
+                then
+                    xtest "\$2" \$SNAP_COMMON/lib/optee_armtz
+                elif [ "\$2" == "-t" ]
+                then
+                    xtest "\$2" "\$3" "\$4"
```

Comment:
Not sure if using `xtest "$@"` would be better and easier to maintain?

Reply:
If we are going to allow any arguments passed to xtest, `xtest $@` would be fine. We can let xtest to check those arguments later. Otherwise, we could use `if` or `case` statements to filter what parameters are supported.

Reply:
I guess if running all arguments that passed with "-f" is doable. And that would be something like below, since `xtest` always input by `command: bin/wrapper xtest`
```shell
elif [ "\$2" == "-t" ]
then
     "\$@"
```